main/IconButton/stories/components/Colors.tsx

```diff
@@ -0,0 +1,62 @@
+import {View} from 'react-native';
+import React from 'react';
+import type {FC} from 'react';
+import type {ThemeType} from '@dooboo-ui/theme';
+import {ThemeProvider} from '@dooboo-ui/theme';
+import {IconButton} from '../../index';
+import {Icon} from '../../../Icon';
+import {Section} from '../../../GlobalStyles';
+import {StyledIcon} from '../../../Icon/stories/Styles';
+
+const IconButtonColors: FC<{themeType: ThemeType}> = ({themeType}) => {
+  return (
+    <ThemeProvider initialThemeType={themeType}>
+      <Section>
+        <IconButton
+          color="primary"
+          size="small"
+          icon="cross-light"
+          style={{padding: 4}}
+        />
+        <IconButton
+          color="secondary"
+          size="small"
+          icon="cross-light"
+          style={{padding: 4}}
+        />
+        <IconButton
+          color="success"
+          size="small"
+          icon="cross-light"
+          style={{padding: 4}}
+        />
+        <IconButton
+          color="warning"
+          size="small"
+          icon="cross-light"
+          style={{padding: 4}}
+        />
+        <IconButton
+          color="danger"
+          size="small"
+          icon="cross-light"
+          style={{padding: 4}}
+        />
+        <IconButton
+          color="info"
+          size="small"
+          icon="cross-light"
+          style={{padding: 4}}
+        />
+        <IconButton
+          color="light"
+          size="small"
+          icon="cross-light"
+          style={{padding: 4}}
+        />
```

Comment:
I'd prefer to define data on repeated elements.
```
const colors = ['light', 'info' ...];
```
Then map above.

Reply:
Additionally, this can be defined easier with [ts-transformer-enumerate](https://www.npmjs.com/package/ts-transformer-enumerate). You may install this in `package.json` if you'd like!